index.html

```diff
@@ -219,14 +229,15 @@ <h2>HTML</h2>
                 <h2>CSS</h2>
 
             </li>
+        </div>
             <li class="frameInfo">Frameworks
                 <p>></p>
             </li>
             <li class="frameInfo">Skills
                 <p>></p>
             </li>
         </ul>
-        </ul>
+        
```

Comment:
- [ ] Please make the language, skills, and framework sections to be dropdown menus. Take a look at this [article](https://www.freecodecamp.org/news/html-drop-down-menu-how-to-add-a-drop-down-list-with-the-select-element/) from free code camp to learn about drop-down menus